ui/app/clinical/consultation/views/treatmentSections/addTreatment.html

```diff
@@ -35,7 +35,7 @@ <h2 class="section-title has-toggle" toggle="addTreatment">
                             active: treatment.isNonCodedDrug
```

Comment:
Why don't we issue a pull request for this right away ? looks like a change product would want to have.